src/components/swap/SwapModalHeader.tsx

```diff
@@ -38,13 +39,15 @@ export default function SwapModalHeader({
           amount={trade.inputAmount}
           currency={inputCurrency ?? trade.inputAmount.currency}
           usdAmount={fiatValueInput.data}
+          isLoading={isPreviewTrade(trade) && trade.tradeType === TradeType.EXACT_OUTPUT}
```

Comment:
q: why doesn't this logic work if it's an EXACT_IN ?

Reply:
if its exact_in, the input amount will not be loading based on the quote response since that is the actual input amount